src/app/components/register/register.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="container">
+  <img
+    class="image"
+    src="https://blog.vantagecircle.com/content/images/2022/06/motivacion-laboral-.png"
+    alt="MotivationApp"
+  />
```

Comment:
This part seems pretty much like the login page.
Let's implement a wrapper component with the common parts.